Dockerfile

```diff
@@ -79,10 +79,10 @@ RUN set -x \
 
 FROM base AS swagger
 # Install go-swagger for validating swagger.yaml
-ENV GO_SWAGGER_COMMIT c28258affb0b6251755d92489ef685af8d4ff3eb
+ENV GO_SWAGGER_COMMIT 5793aa66d4b4112c2602c716516e24710e4adbb5
```

Comment:
If we decide to merge this; perhaps we should tag the commit (also wondering if the fork should live in the `moby` org, so that it's not tied to a personal account)